src/jshint.js

```diff
@@ -707,13 +707,15 @@ var JSHINT = (function() {
           case "7":
           case "8":
           case "9":
+          case "10":
             state.option.moz = false;
             state.option.esversion = +val;
             break;
           case "2015":
           case "2016":
           case "2017":
           case "2018":
+          case "2019":
```

Comment:
side note: I wonder if we should do something similar to babel, marking individual features as allowed/disallowed, and having modules containing presets based on spec versions. These numbers mean less each year in the new living standard model.

Reply:
@caitp When you say "modules," do you mean "npm modules" (i.e. published separately) or "Node.js modules" (i.e. published within this project)?